fontforgeexe/freetypeui.h

```diff
@@ -0,0 +1,74 @@
+#ifndef FONTFORGE_FONTFORGEEXE_FREETYPEUI_H
+#define FONTFORGE_FONTFORGEEXE_FREETYPEUI_H
+
+#include <fontforge-config.h>
```

Comment:
I had a look at the other files in fontforge and most of them are using brackets instead of quotes... Maybe one could go and harmonize them?